### PR TITLE
Fix dark mode colors

### PR DESCRIPTION
**Pull request by MarcelBochtler**
Fixes the issue that text was not readable on multiple UI components when using MacOS's dark system setting.

Before:
SCR-20220211-4c

After:
SCR-20220211-4u

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -127,6 +127,9 @@ compose.desktop {
 
             macOS {
                 iconFile.set(iconsRoot.resolve("icon.icns"))
+                jvmArgs(
+                    "-Dapple.awt.application.appearance=system"
+                )
```
Comment (sschuberth) [inline build.gradle.kts:133]: Nits:
- As long as this is a single argument, should we write it in a single line?
- Commit message: It's "macOS", not "MacOS" 😉

File: src/main/kotlin/ui/Content.kt [hunk 2]
```diff
@@ -1,7 +1,6 @@
 package org.ossreviewtoolkit.workbench.ui
 
 import androidx.compose.runtime.Composable
-
```
Comment (sschuberth) [inline src/main/kotlin/ui/Content.kt:4]: Unrelated change.

**Review by sschuberth — CHANGES_REQUESTED** [review]

File: build.gradle.kts (hunk 1 above)
Comment (MarcelBochtler) [reply build.gradle.kts]: Yes, but I wanted to start the commit message with an upper case letter.
Changed it.

**Review by mnonnenmacher — APPROVED** [review]
